content/desktop/install/msi/install-and-configure.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Install and configure
+description: Understand how to use the MSI installer. Also explore additional configuration options.
+keywords: msi, windows, docker desktop, install, deploy, configure
+---
+
+## Install 
+
+{{< tabs >}}
+{{< tab name="Install interactively" >}}
+
+TODO
```

Comment:
I think for this tab we can mostly copy what already exists.. and for some extra clarity, [this notion page](https://www.notion.so/dockerinc/MSI-v1-Screens-For-Review-dbf0823c6cf24bdaa69d901e088d6c61) has all of the screens for the install.

1. Download the installer using the download button at the top of the page, or from the [download page](TODO).
2. Double-click DockerDesktop.msi to run the installer. By default, Docker Desktop is installed at `C:\Program Files\Docker\Docker`.

3. From the Installation options dialog, you can configure:

  - Create a desktop shortcut

  - Set the Docker Desktop service startup type to automatic

  - Disable Windows Container usage for this installation

  - The engine for Docker Desktop. Either WSL or Hyper-V

If your system only supports one of the two options, you will not be able to select which backend to use.

Follow the instructions on the installation wizard to authorize the installer and proceed with the install.

When the installation is successful, select Finish to complete the installation process.

If your admin account is different to your user account, you must add the user to the docker-users group:

1. Run Computer Management as an administrator.
2. Navigate to Local Users and Groups > Groups > docker-users.
3. Right-click to add the user to the group.
4. Sign out and sign back in for the changes to take effect.